package-lock.json

```diff
@@ -3375,14 +3375,6 @@
         }
       }
     },
-    "@react-native-async-storage/async-storage": {
-      "version": "1.17.6",
-      "resolved": "https://registry.npmjs.org/@react-native-async-storage/async-storage/-/async-storage-1.17.6.tgz",
-      "integrity": "sha512-XXnoheQI3vQTQmjphdXNLTmtiKZeRqvI8kPQ25X5Eae7nZjdYEEGN+0z8N2qyelbUIQwKgmW0aagJk56q7DyNg==",
-      "requires": {
-        "merge-options": "^3.0.4"
-      }
-    },
```

Comment:
Can you check why was this removed?
Better yet, why did we have this dependency since this is not a react native environment

Reply:
This was removed precisely because it was not needed. See https://github.com/Unleash/unleash-proxy-client-js/pull/76

The storage now defaults to `window.localStorage`